Aztec/Classes/NSAttributedString/Conversions/AttributedStringSerializer.swift

```diff
@@ -244,14 +244,15 @@ private extension AttributedStringSerializer {
             return inheritedAttributes
         }
 
+        // ^ Since LI is handled by the OL and UL formatters, we can safely ignore it here.
+        let ignoredElements: [StandardElementType] =  [.li, .figure, .figcaption]
```

Comment:
The comment here seems to be outdated in relation to the code, it could use an update ;)